2000-11-28  Richard Henderson  <rth@redhat.com>

	* com.c (ffecom_member_phase2_): Set TREE_USED on the debugging decl.

--- gcc/f/com.c.jj	Mon Jul 24 11:04:22 2000
+++ gcc/f/com.c	Mon Feb 26 18:54:58 2001
@@ -7127,6 +7127,7 @@ ffecom_member_phase2_ (ffestorag mst, ff
   TREE_STATIC (t) = TREE_STATIC (mt);
   DECL_INITIAL (t) = NULL_TREE;
   TREE_ASM_WRITTEN (t) = 1;
+  TREE_USED (t) = 1;
 
   DECL_RTL (t)
     = gen_rtx (MEM, TYPE_MODE (type),
